.github/workflows/docker-release.yml

```diff
@@ -90,7 +90,7 @@ jobs:
 
         go test ./contrib/charts/dragonfly/... -update
 
-        git commit \
+        git commit --allow-empty \
```

Comment:
maybe instead of comitting an empty commit we can skip the commit? will `|| true` work for this situation?

Reply:
Yep, should work. updated.